.github/workflows/build.yml

```diff
@@ -40,29 +40,20 @@ jobs:
           sudo apt install pandoc
           pip install -U pip setuptools
           pip install -r requirements.txt
-          sphinx-build -W -a docs/reference/ docs/reference/_build/html
-  build-tutorial-documentation:
-    runs-on: ubuntu-latest
-    continue-on-error: true
-    steps:
-      - uses: actions/checkout@v2
-      - name: Set up Python 3.8
-        uses: actions/setup-python@v1
+          pip freeze > versions_at_documentation_build.txt
+          sphinx-build -W -t build_tutorials -a docs/ docs/_build/html
+      - name: Archive documentation version artifact
+        uses: actions/upload-artifact@v2
         with:
-          python-version: 3.8
-      - name: "Run Tutorial Documentation Generation"
-        run: |
-          sudo apt install pandoc
-          pip install -U pip setuptools
-          pip install -r requirements.txt
-          sphinx-build -W -a docs/tutorials/ docs/tutorials/_build/html
-          touch docs/tutorials/_build/html/.nojekyll
-      - name: Archive production artifacts
+          name: versions_at_documentation_build
+          path: |
+            versions_at_documentation_build.txt
+      - name: Archive documentation artifacts
         uses: actions/upload-artifact@v2
```

Comment:
does this zip it up too, or are these actually uploading a "folder" artifact?

Reply:
My understanding is that the file is uploaded individually but the build artifacts UI only allows you to download it as a ZIP of the text file.  In my tests it worked as expected and I could download the zip of the file.